README.md

```diff
@@ -1,19 +1,31 @@
 # Taxonomy Editor
-Taxonomies are at the 🧡 of Open Food Facts data structure. This project provides an user-friendly editor for editing taxonomies easily.
 
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://static.openfoodfacts.org/images/logos/off-logo-horizontal-dark.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://static.openfoodfacts.org/images/logos/off-logo-horizontal-light.svg">
+  <img height="90" src="https://static.openfoodfacts.org/images/logos/off-logo-horizontal-light.svg">
```

Comment:
Also code seems correct, this does not work for me ! 

![image](https://user-images.githubusercontent.com/144788/202128212-d7e2b2f6-3932-48b0-8e49-6295cdb91a8c.png)

I would say it's because those are external images and github seems to transform them to camo.github … url, screwing them in the process.

So maybe the simplest is to include those svg images in the repository (in the doc/ folder)

Reply:
Yep @alexgarel, added it. Checked it with light mode too. 

![Screenshot](https://user-images.githubusercontent.com/25586296/202164901-30d2e3b4-77b6-42f8-9b12-996039d17501.png)